android/src/main/java/com/swmansion/rnscreens/ScreenContainer.java

```diff
@@ -235,8 +235,12 @@ private void detachScreen(ScreenFragment screenFragment) {
     getOrCreateTransaction().remove(screenFragment);
   }
 
+  protected Screen.ActivityState getActivityState(ScreenFragment screenFragment) {
+    return screenFragment.getScreen().getActivityState();
+  }
+
   protected boolean isScreenActive(ScreenFragment screenFragment) {
-    return screenFragment.getScreen().isActive();
+    return getActivityState(screenFragment) != Screen.ActivityState.INACTIVE;
```

Comment:
I think it'd be better to abandon this method altogether. We only use it in `performUpdate` where one out of three calls have been replaced with direct comparison of activity state. We should do the same with two remaining calls. This method is not used outside + I think that using term "active" is going to cause more confusion